### ISSUE TITLE
Question: Command line example on running your code

### ISSUE THREAD

**Issue by hql5143** [ecs-vlc/FMix#13]
Hi 

Could you provide a sample command line to run your code on a dataset such as CIFAR-10 or -100?

**Comment by ethanwharris** [comment]
Hi, thanks for the issue!

We'll have some training examples and pre-trained models up in the next couple of days.
I'll post here when it's done :)

**Comment by ethanwharris** [comment]
@hql5143 I've added some more guidance to the README under [Experiments](https://github.com/ecs-vlc/FMix#experiments), let us know if you have any issues getting it to run :)

**Comment by hql5143** [comment]
Thank you much! Following the instructions, I was able to run the experiments!!

A follow-up question: May I ask if "bash cifar_experiment.sh cifar10 pyramidnet fmix ./data"
 would yield something close to the reported accuracy: 98.64?

Also, what would be the batch size for "pyramidnet"; it seems not specified in the bash file...

**Comment by ethanwharris** [comment]
@hql5143 it should do :) the batch size is 64 ~~just the default of 128 listed in `trainer.py`~~.

Note that the PyramidNet experiments use the same schedule as Fast AutoAugment (1800 epochs) - they take about 8days on a V100!

Another option is to load in the pretrained model from torch.hub :)

Hope that helps!
